staging/src/k8s.io/apimachinery/pkg/apis/meta/v1/micro_time.go

```diff
@@ -175,10 +175,10 @@ func (t *MicroTime) Fuzz(c fuzz.Continue) {
 	if t == nil {
 		return
 	}
-	// Allow for about 1000 years of randomness.  Leave off nanoseconds
-	// because JSON doesn't represent them so they can't round-trip
-	// properly.
-	t.Time = time.Unix(c.Rand.Int63n(1000*365*24*60*60*1000*1000), 0)
+	// Allow for about 1000 years of randomness. Accurate to a tenth of
+	// micro second. Leave off nanoseconds because JSON doesn't
+	// represent them so they can't round-trip properly.
+	t.Time = time.Unix(c.Rand.Int63n(1000*365*24*60*60), 1000*c.Rand.Int63n(1000000))
```

Comment:
Then why not set the second param to 0.

Reply:
The first param is `second`. The 2nd param is `nonosecond`. And 1 micro second = 1000 nano second. 
We do not set the second param to 0, because we need a random value for micro second.

Let me provide two examples:
When we encode time.Unix(xxxxxx, 1008) to json, then decode it back what will happen?
The time will become time.Unix(xxxxxx, 1000), the second param is `lost`.

When we encode time.Unix(xxxxxx, 2000) to json, then decode it back what will happen?
The time is still time.Unix(xxxxxx, 2000)

So the second param can be values like 1000, 2000, 999000.
But it can't be 1, 1001, 999001.